src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx

```diff
@@ -55,7 +55,6 @@ export const ColorPickerColorsGrid = React.forwardRef(
               colorSize={colorSize}
               tooltipContent={tooltipContentByColor[color]}
               colorShape={colorShape}
-              data-testid={`color-picker-item-${color}`}
```

Comment:
@hadasfa internally, the component is setting its test id to `dataTestId || getTestId`. I thought it will make more sense but I don't have a strong opinion - do you want me to change it?